.flake8

```diff
@@ -8,7 +8,7 @@
 # W504: line break after binary operator
 #       (Raised by flake8 even when it is followed)
 ignore = E126, E402, E129, W504
-max-line-length = 152
+max-line-length = 153
```

Comment:
It was modified from 154 to 153, then i pushed to the repo and initiated a pull request but got a conflict. So I think the conflict was triggered by duplicates line of code along with the conflict markers, so i just deleted one line and the markers because you need to decide which version of the max-line-length setting you want to keep. Since both versions are adjacent and come from the same branch (master), you can simply choose one of them.

Reply:
*nod; I understand.  I'm asking you to think critically.  What is the point of the ticket?  And, more precisely, what is the specific instruction?

When that is clear, what needs to happen here?  I understand the steps you've taken, but now what would otherwise be merged is not accurate to the ticket.

Reply:
Making the line length smaller, then I will test and validate. I will update my pull request. What do you think please?

Reply:
Hi @khk-globus,

I wanted to inform you that I have committed and pushed the latest changes  to my repository.

Please review the updates at your convenience, and please let me know if you have any feedback or questions.

Reply:
Thank you; FWIW ("for what it's worth"), you don't need to explicitly tell me this type of detail.  When one enters a PR conversation (e.g., making the PR, commenting on the PR), GitHub will email them thereafter.

Reply:
> Please review the updates at your convenience, and please let me know if you have any feedback or questions.

Now the hunk to change the line-length is missing entirely.  Again, what is the point of the ticket?  What is the specific task for the PR?

Reply:
@khk-globus Please what do you suggest I do.

Reply:
>> *nod; I understand. I'm asking you to think critically. What is the point of the ticket? And, more precisely, what is the specific instruction?

> @khk-globus Please what do you suggest I do.

This is the "think critically" part.  I know what _I_ might do, but that's not important.  This is your pull request, not mine.  A couple of thoughts come to mind:

- What does GitHub show will be changed if this PR is merged?  (Hint: "Files changed" tab)
- What does issue #3105 state to do?

Do you see a discrepancy?

Reply:
Thank you @khk-globus, This shows that my PR isnt Aligned with the goals of the Issue #3105